frontends/web/src/routes/device/bitbox02/bitbox02.tsx

```diff
@@ -330,12 +316,12 @@ class BitBox02 extends Component<Props, State> {
         title: this.props.t('bitbox02Interact.confirmDate'),
         text: this.props.t('bitbox02Interact.confirmDateText'),
       } });
-      apiPost('devices/bitbox02/' + this.props.deviceID + '/backups/create').then(({ success }) => {
-        if (!success) {
+
+      createBackup(this.props.deviceID)
+        .then(() => this.setState({ creatingBackup: false, waitDialog: undefined }))
+        .catch(() => {
           alertUser(this.props.t('bitbox02Wizard.createBackupFailed'), { asDialog: false });
-        }
-        this.setState({ creatingBackup: false, waitDialog: undefined });
-      });
+        });
```

Comment:
using promise.reject if success is false so we can use .catch but I don't think it is better, mainly because the passed argument cannot be typed.